_i18n/en.yml

```
@@ -818,29 +819,32 @@ roadmap:
   onionaddress: Onion address for Getmonero.org
   supercop: Wallet scanning speedups (support for supercop ASM)
   clsag: Concise linkable spontaneous anonymous group (CLSAG) signatures
-  tryptych: "Triptych: logarithmic-sized linkable ring signatures with applications"
   kastelo: "Kastelo: open source hardware wallet"
-  layer2: Second-layer solutions for speed and scalability
   released-0-16-0-0: CLI and GUI 0.16.0.0 released
   released-0-17: CLI and GUI 0.17 'Oxygen Orion' released
   
-  returnaddr: Return addresses
   atomicswaps: Monero <-> Bitcoin atomic swaps
   bplus: Bulletproofs+
   p2pool: P2Pool released
   haveno: Haveno
 
-  ringsize-sera-tryp: increase ring size to >100
   seraphis-jamtis: Seraphis & Jamtis
```

```suggestion
  seraphis-jamtis: Seraphis codebase & Jamtis
```
this small but important change will reflect the projects change in direction. i can than ask rbrunner to clarify this at the top of the wiki we link to. this comes from some comment kayabanerve left in #monero-research-lounge. ill post logs below